ols/app/endpoints/ols.py

```diff
@@ -46,10 +71,18 @@ def conversation_request(llm_request: LLMRequest) -> LLMRequest:
     logger.info(f"{conversation} Incoming request: {llm_request.query}")
 
     # Validate the query
-    question_validator = QuestionValidator()
-    validation_result = question_validator.validate_question(
-        conversation, llm_request.query
-    )
+    try:
+        question_validator = QuestionValidator(
+            provider=llm_request.provider, model=llm_request.model
+        )
+        validation_result = question_validator.validate_question(
+            conversation, llm_request.query
+        )
+    except LLMConfigurationError as e:
```

Comment:
that's ok for now, but we need to be super careful what info is in this exception. I.e. no BAM keys etc.